dna/metrics.py

```diff
@@ -45,5 +45,8 @@ def top_k_regret(ranked_data: pd.DataFrame, actual_data: pd.DataFrame, k: int):
 
 def spearman_correlation(ranked_data: pd.DataFrame, actual_data: pd.DataFrame):
     actual_data = pd.DataFrame(actual_data)
-    score = scipy.stats.spearmanr(ranked_data['rank'], utils.rank(actual_data.test_f1_macro))
+    ranked_data = ranked_data['rank']
+    actual_data = utils.rank(actual_data.test_f1_macro)
+    score = scipy.stats.spearmanr(ranked_data, actual_data)
```

Comment:
Try to keep the changes in a single PR scoped as much as possible. This change is not related to the topic of this PR, so it makes it harder to review and potentially longer before it gets merged. For example, as the reviewer, if I approved of everything in the PR except this unrelated changed, I would not merge the PR.

Reply:
This change is a result of incorporating plotting into the score function and then separating it from the score function. At this point, I don't remember exactly what the function definition was before but I can copy it from a previous commit.

Reply:
Yes, just look at the git history.